stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -331,6 +333,66 @@ private void getStyleFromFloat(
     styles.store(name, String.valueOf(value), isDefaultValue(value));
   }
 
+  private void getStyleFromObject(
+      View view,
+      String name,
+      Object value,
+      @Nullable ViewDebug.ExportedProperty annotation,
+      StyleAccumulator styles) {
+    if (annotation != null && annotation.deepExport() && value != null) {
```

Comment:
consider inverting this conditional so that the body of the method isn't indented, like so ...

```
if (annotation == null || !annotation.deepExport() || value == null) {
  return;
}
```